src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerViewModel.cs

```diff
@@ -1,5 +1,9 @@
 using Dynamo.ViewModels;
-using Dynamo.Core;
+#if NETFRAMEWORK
+using NotificationObject = Microsoft.Practices.Prism.ViewModel.NotificationObject;
+#else
+using NotificationObject = Dynamo.Core.NotificationObject;
```

Comment:
Can you explain again why `Dynamo.Core.NotificationObject` doesn't work in .net48?

Reply:
replacing all Microsoft.Practices.Prism.ViewModel.NotificationObject with Dynamo.Core.NotificationObject might cause breaking changes in the public API.
From the little testing I did in DynamoRevit it looked fine, but I cannot be sure about other Dynamo API use cases.